src/molecules/StarRating/Readme.md

```diff
@@ -0,0 +1,7 @@
+### StarRating Example:
+
+```jsx
+    <StarRating>
+      <strong>New Component Created!</strong><br /><small>Now go and write some code! -- <em>You code monkey, you ;)</em></small>.
+    </StarRating>
```

Comment:
@Jexeones24 Please add an example here of a `StarRating` component with its props so the next dev will know how it works.